app/facades/admin/pages/new_facade.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Admin
+  module Pages
+    class NewFacade
+      attr_reader :page
+
+      ACTION = 'new'
+
+      def initialize(page: Page.new)
+        @page = page
+      end
+
+      def sections_for_select
+        Section.widgets.map { |k, v| [k.humanize.upcase, v] }
+      end
+
+      def edit_section_link_show?(object, action)
+        object.id.present? && action == ACTION
```

Comment:
that's possible that I'm missing the point of these two methods, but operating with strings as keys is a bad approach anyway. 

` strings as keys ` == decide the way method gonna go by passing string literals (it's Sunday and it's difficult to talk and think, give me a break :) )